### ISSUE TITLE
Training the best model from scratch

### ISSUE THREAD

**Issue by lukeN86** [SLDGroup/ZiCo#7]
Hello,

thank you for your great work!

Could you please verify/share the exact hyperparameter setting to train the 1G FLOPS model?

I ran the training part of the `scripts/ZiCo_NAS_ImageNet_flops1G.sh` script - I referenced directly the `ZiCo/ZiCo_imagenet1k_flops1G_res224.txt` architecture to avoid any randomness coming from the search algorithm - however the accuracy I get on ImageNet1k is only 80.1% +- 0.1 (I tried 3 different runs for 480 epochs). When using the checkpoint you provide, I get the reported 80.5% accuracy, so there is no problem there - so I'm wondering whether there is perhaps some other parameter I should change in the `ZiCo_NAS_ImageNet_flops1G.sh` script? 

The training script I used is exactly like the one you provided, with only the change being the reference to your architecture (the `--plainnet_struct_txt` param)
```
resolution=224
epochs=480

horovodrun -np 8 python ts_train_image_classification.py --dataset imagenet --num_classes 1000 \
  --dist_mode single --workers_per_gpu 8 \
  --input_image_size ${resolution} --epochs ${epochs} --warmup 5 \
  --optimizer sgd --bn_momentum 0.01 --wd 4e-5 --nesterov --weight_init custom \
  --label_smoothing --random_erase --mixup --auto_augment \
  --lr_per_256 0.1 --target_lr_per_256 0.0 --lr_mode cosine \
  --arch Masternet.py:MasterNet \
  --plainnet_struct_txt ZiCo/ZiCo_imagenet1k_flops1G_res224.txt \
  --teacher_arch geffnet_tf_efficientnet_b3_ns \
  --teacher_pretrained \
  --teacher_input_image_size 320 \
  --teacher_feature_weight 1.0 \
  --teacher_logit_weight 1.0 \
  --ts_proj_no_relu \
  --ts_proj_no_bn \
  --use_se \
  --target_downsample_ratio 16 \
  --batch_size_per_gpu 64 --save_dir ${save_dir} \
  --world-size 8 \
  --dist_mode horovod \
  --auto_resume \
```

Output

```

argv=
['ts_train_image_classification.py', '--dataset', 'imagenet', '--num_classes', '1000', '--dist_mode', 'single', '--workers_per_gpu', '8', '--input_image_size', '224', '--epochs', '480', '--warmup', '5', '--optimizer', 'sgd', '--bn_momentum', '0.01', '--wd', '4e-5', '--nesterov', '--weight_init', 'custom', '--label_smoothing', '--random_erase', '--mixup', '--auto_augment', '--lr_per_256', '0.1', '--target_lr_per_256', '0.0', '--lr_mode', 'cosine', '--arch', 'Masternet.py:MasterNet', '--plainnet_struct_txt', 'ZiCo/ZiCo_imagenet1k_flops1G_res224.txt', '--teacher_arch', 'geffnet_tf_efficientnet_b3_ns', '--teacher_pretrained', '--teacher_input_image_size', '320', '--teacher_feature_weight', '1.0', '--teacher_logit_weight', '1.0', '--ts_proj_no_relu', '--ts_proj_no_bn', '--use_se', '--target_downsample_ratio', '16', '--batch_size_per_gpu', '64', '--save_dir', './save_dir/ZiCo_NAS_ImageNet_flops1G_480ep_docker_8gpu', '--world-size', '1', '--dist_mode', 'horovod']
opt=
Namespace(evaluate_only=False, apex=False, apex_loss_scale='dynamic', apex_opt_level='O1', fp16=False, dist_mode='horovod', independent_training=False, world_size=8, rank=0, gpu=0, sync_bn=False, num_job_splits=None, job_id=None, fp16_allreduce=False, batches_per_allreduce=1, lr=0.2, target_lr=0.0, lr_per_256=0.1, target_lr_per_256=0.0, lr_mode='cosine', warmup=5, epoch_offset=0.0, lr_stage_list=None, lr_stage_decay=None, optimizer='sgd', momentum=0.9, adadelta_rho=0.9, adadelta_eps=1e-09, weight_decay=4e-05, resume=None, auto_resume=False, load_parameters_from=None, strict_load=False, start_epoch=0, epochs=480, save_dir='./save_dir/ZiCo_NAS_ImageNet_flops1G_480ep_docker_8gpu', save_freq=10, print_freq=100, label_smoothing=True, weight_init='custom', nesterov=True, grad_clip=None, bn_momentum=0.01, bn_eps=None, mixup=True, random_erase=True, auto_augment=True, no_data_augment=False, data_dir=None, dataset='imagenet', workers_per_gpu=8, batch_size=512, batch_size_per_gpu=64, auto_batch_size=False, num_cv_folds=None, cv_id=None, input_image_size=224, input_image_crop=0.875, arch='Masternet.py:MasterNet', pretrained=False, num_classes=1000, dataloader_testing=False, teacher_input_image_size=320, teacher_arch='geffnet_tf_efficientnet_b3_ns', teacher_pretrained=True, ts_proj_no_relu=True, ts_proj_no_bn=True, teacher_load_parameters_from=None, teacher_feature_weight=1.0, teacher_logit_weight=1.0, ts_clip=None, target_downsample_ratio=16)
-----
Create model: Masternet.py:MasterNet
---debug use_se in SuperResIDWE1K7(48,48,2,72,1)
---debug use_se in SuperResIDWE1K7(48,80,2,128,1)
---debug use_se in SuperResIDWE2K7(80,160,2,144,3)
---debug use_se in SuperResIDWE4K7(160,288,2,232,4)
---debug use_se in SuperResIDWE6K5(288,232,1,208,5)
Use custom init
loading student_model:
SuperConvK3BNRELU(3,48,2,1)SuperResIDWE1K7(48,48,2,72,1)SuperResIDWE1K7(48,80,2,128,1)SuperResIDWE2K7(80,160,2,144,3)SuperResIDWE4K7(160,288,2,232,4)SuperResIDWE6K5(288,232,1,208,5)SuperConvK1BNRELU(232,2048,1,1)


...

Eval epoch=479, i=0, loss=0.413227, acc1=93.75%, acc5=98.4375%
14859  * Validate Acc@1 80.114 Acc@5 95.134, n_val=50000.0
14860 --- Epoch=479, Elasped hour=   142.6, Remaining hour=  0.2971, Training Speed=1229.59, best_acc1=80.138, best_acc1_at_epoch=476, best_acc5=95.14199829101562, best_acc5_at_epoch=478

```
